en/docs/guides/identity-lifecycles/extend-username-and-password-recovery.md

```diff
@@ -0,0 +1,534 @@
+      "resendCode": "8dde8fd4-c58d-4408-a835-a9954ebc278a",
+      "links": [
+         {
+            "rel": "next",
+            "href": "/t/carbon.super/api/users/v1/recovery/password/confirm",
+            "type": "POST"
+         },
+         {
+            "rel": "resend",
+            "href": "/t/carbon.super/api/users/v1/recovery/password/resend",
+            "type": "POST"
+         }
+      ]
+    }
+    ```
+
+3. If you want to resend the notifications to the user via the notified channel in the above step, use the `resendCode` with the resend notifications API.
+
+    ```java tab="Request"
+    curl -X POST "https://localhost:9443/api/users/v1/recovery/password/resend" -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"resendCode\":\"1234-2ws34-1234\",\"properties\":[{\"key\":\"key\",\"value\":\"value\"}]}"
+    ```
+
+    ```java tab="Response"
+    {
+    "code": "UAR-02001",
+    "message": "Confirmation code resent to the user.",
+    "notificationChannel": "EMAIL",
+    "resendCode": "8ebefae5-0a80-4edf-ac2d-6034384e45c0",
+    "links": [
+        {
+            "rel": "next",
+            "href": "/t/carbon.super/api/users/v1/recovery/password/confirm",
+            "type": "POST"
+        },
+        {
+            "rel": "resend",
+            "href": "/t/carbon.super/api/users/v1/recovery/password/resend",
+            "type": "POST"
+        }
+    ]
+    }
+    ```
+
+4. Use the confirmation code received by the user to verify the confirmation code.
+
+    ```java tab="Request"
+    curl -X POST "https://localhost:9443/api/users/v1/recovery/password/confirm" -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"confirmationCode\":\"1234-2ws34-12345\",\"properties\":[{\"key\":\"key\",\"value\":\"value\"}]}"
+    ```
+
+    ```java tab="Response"
+    {
+        "resetCode": "90b9ce11-7642-4f50-aa06-386011b7de66",
+        "links": [
+            {
+                "rel": "next",
+                "href": "/t/carbon.super/api/users/v1/recovery/password/reset",
+                "type": "POST"
+            }
+        ]
+    }
+    ```
+
+5. Use the `resetCode` and the new password to update the existing password and recover the account.
+
+    ```java tab="Request"
+    curl -X POST "https://localhost:9443/api/users/v1/recovery/password/reset" -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"resetCode\":\"aefaef12-951e-4a42-b01b-3118798f58c3\",\"password\":\"newPassword\",\"properties\":[{\"key\":\"key\",\"value\":\"value\"}]}"
+    ```
+
+    ```java tab="Response"
+    {
+        "code": "PWR-02005",
+        "message": "Successful password reset."
+    }
+    ```
+
+### With external notification management
+
+1. Use the following command to create a username recovery request.
+
+    ```java tab="Request"
```

Comment:
We must update these Request/Response code samples to the following format.

https://is.docs.wso2.com/en/6.0.0/guides/login/oidc-basic-client-profile/#try-out-the-flow